plugins/services/src/js/components/ServiceStatusProgressBar.js

```diff
@@ -0,0 +1,81 @@
+import React, { PropTypes } from "react";
+import { Tooltip } from "reactjs-components";
+
+import StringUtil from "#SRC/js/utils/StringUtil";
+import ProgressBar from "#SRC/js/components/ProgressBar";
+import Pod from "../structs/Pod";
+import Service from "../structs/Service";
+import ServiceTree from "../structs/ServiceTree";
+import ServiceStatus from "../constants/ServiceStatus";
+
+class ServiceStatusProgressBar extends React.Component {
+  getTooltipContent() {
+    const { service } = this.props;
+    const { tasksRunning } = service.getTasksSummary();
+    const instancesTotal = service.getInstancesCount();
+
+    return (
+      <div className="tooltip-line-item">
+        <span>
+          {`${tasksRunning} ${StringUtil.pluralize("instance", tasksRunning)} running out of ${instancesTotal} `}
+        </span>
+      </div>
+    );
+  }
+
+  render() {
+    const { service } = this.props;
+    const instancesCount = service.getInstancesCount();
+    const serviceStatus = service.getStatus();
+    const tasksRunning = service.getTaskCount();
+
+    if (
+      serviceStatus === ServiceStatus.RUNNING.displayName ||
+      serviceStatus === ServiceStatus.STOPPED.displayName
+    ) {
+      return null;
+    } else if (serviceStatus === ServiceStatus.DELETING.displayName) {
```

Comment:
I think we like to fully flatten these kind of `if else` blocks.

```
if (RUNNING || STOPPED) {
   return ...
}

if (DELETING) {
  return ...
}
```

Reply:
why?

Reply:
I've been suggested this in PRs recently. And I've been convinced that it's the simpler of the two options because it makes it very clear that each block is completely independent.

Reply:
To @ahoskins point - you don't need the ` else if (` here because if it fell into the `if` statement above it, it would return anyway. IMO it's much cleaner to read multiple, separate `if` statements than a series of `if else` statements.

Reply:
Let me add an example where an else if is needed or better used:

```JS
if (x > 1000) {
  ...
} else if (x > 100) {
  ...
} else if (x > 10) {
  ... 
}

return y;
```
In that case only one block will be executed but all blocks do have the same return that is a good example for a else if structure. Because each condition is simpler to read then x is bigger then z and smaller then z. But all conditions are connected to each other. 

But if your condition where `x === 3` ,`x  === 2`, `x === 1` you would most certainly not write your condition like `x === 3 && x !== 2 && x !== 1` so in that case it makes more sense to actually split up the if blocks as there are not related to each other. Each condition stands and works for it self. 

In this case although a switch would also work very fine:
```JS
switch(serviceStatus) {
  case ServiceStatus.RUNNING.displayName:
  case ServiceStatus.STOPPED.displayName:
    return null;
  case ServiceStatus.DELETING.displayName:
    ...
  default:
    ...
}
```
But I do think we are trying not to use switch blocks in dcos-ui for the reason that they are difficult to mental parse them. But I could be mistaken.

Reply:
Now I understood what you meant @ahoskins

I thought you were talking about using `RUNNING || STOPPED` instead `ServiceStatus.RUNNING.displayName || ServiceStatus.STOPPED.displayName`

Next time will be easier to just say `no need to use "else if"`, I'm also an advocate for clean if only perhaps switch case statements in other cases, but you will see me advocating that on previous PRs.

Thanks for clarifying @MatApple @Poltergeist